### PR TITLE
[Aviraj] Added blog Build a Donation dApp on Celo to award your Favorite Content Creator

### PR DESCRIPTION
**Pull request by avirajkhare00**
Thanks for submitting this tutorial for Celo Sage
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
 - [x] The title of the Pull Request matches the title of my tutorial.
 - [x] I've followed the tutorial template structure.
 - [x] I've proofread the tutorial for spelling and grammar mistakes.
 - [x] I've thoroughly explained the code, with inline code comments and text explanations.
 - [x] I've run a linter & prettifier to format my code blocks.
 - [x] I've run the tutorial to test it and the code works.

### CODE REVIEW

File: blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+Now we have the clear picture, let's get started...
+
+## Code
+
+### Smart Contract
+
+Below is the smart contract that will faciliate all the features.
+
+```js
```
Comment (developerkunal) [inline blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md:52]: Change ``` js  to solidity ```

Comment (avirajkhare00) [reply blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md:52]: OK

Comment (avirajkhare00) [reply blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md:52]: Markdown doesn't detect `solidity` so it's `js`

Comment (avirajkhare00) [reply blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md:52]: In all my blogs it's `js`

Comment (developerkunal) [reply blog/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator/04-02-2023-build-a-donation-dApp-on-celo-to-award-your-favourite-content-creator.md:52]: @avirajkhare00 it does detect solidity you can check my published posts. https://github.com/celo-org/docs/pull/584

**Review by developerkunal — COMMENTED** [review]
Looks good to me @viral-sangani

**Comment by developerkunal** [comment]
@avirajkhare00 add description to pull request
Thanks for submitting this tutorial for Celo Sage
Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

 My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
 The title of the Pull Request matches the title of my tutorial.
 I've followed the tutorial template structure.
 I've proofread the tutorial for spelling and grammar mistakes.
 I've thoroughly explained the code, with inline code comments and text explanations.
 I've run a linter & prettifier to format my code blocks.
 I've run the tutorial to test it and the code works.
 
 this one

**Comment by avirajkhare00** [comment]
@developerkunal done :)

**Review by viral-sangani — APPROVED** [review]

**Pull request merged**